pkg/framework/context/context.go

```diff
@@ -126,12 +126,14 @@ func (p *Provider) InboundMessageHandler() transport.InboundMessageHandler {
 			}
 		}
 
-		// in case of no services are registered for given message type,
-		// find generic inbound services registered for given message header
-		for _, svc := range p.messageServices {
-			if svc.Accept(msg.Header) {
-				_, err = svc.HandleInbound(msg, myDID, theirDID)
-				return err
+		if p.msgHandler != nil {
```

Comment:
Should be reverted.

Reply:
using default noop version of message service provider if not set.